src/mono/browser/test-main.js

```diff
@@ -312,6 +321,12 @@ async function run() {
         App.runtime = await dotnet.create();
         App.runArgs = runArgs
 
+        if (ENVIRONMENT_IS_WEB && isFirefox) {
```

Comment:
Why this is in `async function run()` when for chrome we do out of any function (line 60)? When logically it does not change much, it's confusing.

Reply:
In chrome it's best to print it as soon as possible, and chrome driver would give it to us. In firefox it doesn't work.